README.md

```diff
@@ -58,7 +58,7 @@ If your project starts with `TestUtilities.` or `!TestUtilities.`, the `NI.TestU
     - [[F.3.1] ❌ **AVOID** Adding excessive complexity in compound statements](#f31--avoid-adding-excessive-complexity-in-compound-statements)
     - [[F.3.2] ✔️ **CONSIDER** Extracting methods from complex nested expressions](#f32-️-consider-extracting-methods-from-complex-nested-expressions)
     - [[F.3.3] ✔️ **CONSIDER** Splitting long parameter lists](#f33-️-consider-splitting-long-parameter-lists)
-    - [[F.3.4] ✔️ **CONSIDER** Splitting long chains of dotted method/property invocations](#f34-️-consider-splitting-long-chains-of-dotted-methodproperty-invocations)
+    - [[F.3.4] ✔️ **DO** Split chains of method invocations with lambda expressions 💻](#f34-️-do-split-chains-of-method-invocations-with-lambda-expressions)
```

Comment:
I don't think this hyperlink is correct. Did you test clicking on F.3.4? I believe you need to add an extra hyphen at the end of the hyperlink to correspond to the computer emoji